proc-macros/tests/ui/incorrect/rpc/rpc_name_conflict.rs

```diff
@@ -0,0 +1,13 @@
+use jsonrpsee::proc_macros::rpc;
+
+// Associated items are forbidden.
+#[rpc(client, server)]
+pub trait MethodNameConflict {
+	#[method(name = "foo")]
+	async fn foo(&self) -> u8;
+
+	#[method(name = "foo")]
```

Comment:
Ok, but this is the easy case. Let's do two separate modules?

Reply:
Separate modules need to be merged on runtime and will return errors, there is no way to check them on compile time.

Reply:
Good pt!